pyproject.toml

```diff
@@ -14,11 +14,11 @@ poetry-plugin-export = ">=1.5.0"
 cleo = ">=2.0.1"
 
 [tool.poetry.group.dev.dependencies]
-black = "^22.8.0"
-behave = "^1.2.6"
-nose = "^1.3.7"
-pylint = "^3.2.2"
-kappa-maki = "^1.0.2"
+black = ">=22.8.0"
+behave = ">=1.2.6"
+nose = ">=1.3.7"
+pylint = ">=3.2.2"
+kappa-maki = ">=1.0.2"
```

Comment:
Thanks for being prudent with these, as well as the comments for temporary pom elements during  the transition.